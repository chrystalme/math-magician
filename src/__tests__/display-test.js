import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

import Display from '../components/display';

describe('Display', () => {
  test('display component is rendered correctly', () => {
    const component = render(
      <Display />,
    );
    expect(component).toMatchSnapshot();
  });
});
