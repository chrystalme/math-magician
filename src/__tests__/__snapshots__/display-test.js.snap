// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display display component is rendered correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        style="font-weight: 700; font-size: x-large; color: tomato; background-color: whitesmoke; display: flex; justify-content: flex-end; position: relative; width: 300px; height: 70px; padding: 5px;"
      >
        0
        <span
          style="color: black; top: 50px; left: 380px;"
        />
        <span />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      style="font-weight: 700; font-size: x-large; color: tomato; background-color: whitesmoke; display: flex; justify-content: flex-end; position: relative; width: 300px; height: 70px; padding: 5px;"
    >
      0
      <span
        style="color: black; top: 50px; left: 380px;"
      />
      <span />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
